packages/circuit-ui/tsconfig.json

```diff
@@ -9,14 +9,13 @@
     "isolatedModules": true,
     "jsx": "react-jsx",
     "jsxImportSource": "@emotion/react",
-    "module": "ES2020",
+    "module": "es2020",
     "moduleResolution": "node",
     "noFallthroughCasesInSwitch": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "outDir": "./dist/es",
     "resolveJsonModule": true,
-    "skipLibCheck": true,
```

Comment:
I don't think we need this, and the TS docs say that skipping the check makes types less safe. Even if it makes our builds slightly longer, I think that's fine